sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,699 @@
 
 - Did this and that ... intel/llvm#pr
 
+# Release notes Mar'25
+
+Release notes for commit range
+[b0212c37b2](https://github.com/intel/llvm/commit/b0212c37b230d9dd3bb129df9f4ecc417b92ad8)
+...
+[b23d69e2c3](https://github.com/intel/llvm/commit/b23d69e2c3fda1d69351137991897c96bf6a586d)
+
+## New Features
+
+### Runtime compilation of SYCL code
```

Comment:
@intel/dpcpp-kernel-fusion-reviewers, tagging you here to review this section

Reply:
@intel/dpcpp-kernel-fusion-reviewers, friendly ping

Reply:
I'm working on a more descriptive summary for the feature.

How was the list of PRs compiled? I think there are a couple of related PRs missing, but I'm also not sure how valuable a complete list is for the release notes.

Reply:
@jopperm thanks! The list is the slice of commits which go to Mar'25 release. However some patches really may not be present here in release notes. The reason is they were merged to the **closed**-source compiler release after **open**-source compiler code cut-off. We align closed-source compiler releases with open-source compiler releases. We will identify these patches later and manually cherry-pick them to the Mar'25 release branch of open-source compiler (this compiler).
If you know some exact patches, let's contact internally to identify if they were committed to closed-source compiler. If they were, we need to include information about them to these release notes